newshomepages/screenshot.py

```diff
@@ -39,159 +60,22 @@ def _screenshot(
-            ".fancybox-overlay-fixed",
-            ".ab_widget_container_popin-image",  # Popover
-            ".ab_widget_container_popin-image_content",
-            ".ab_widget_container_popin-image_overlay",
-            ".fc-ab-root",  # Popover
-            ".modal-backdrop",  # Modal popup
-            "#dth-borderless-modal-alt",
-            "#floatingBlockerGuide",  # Popup overlay
-            ".ctct-popup-overlay",  # Popup overlay
-            ".ctct-popup-wrapper",
-            ".message-container",  # Popup
-            ".teg-container",
-            'div[id^="sp_message_container"]',  # Popover
-            ".wisepops-root",  # Popover
-            "#wisepops-root",
-            ".widget_eu_cookie_law_widget",  # GDPR poup
-            ".fancybox-overlay",  # Ad overlay
-            ".signup-box",  # Popup ad
-            ".newspack-lightbox",  # Popup
-            ".adunit-googleadmanager",  # Google ad
-            ".mc-modal",  # Newsletter popup
-            ".mc-modal-bg",
-            ".blockNavigation",
-            ".ab-iam-root",
-            ".omaha-background",  # Takeover ad
-            ".sqs-popup-overlay",  # Overlay popup
-            "#gcomPromo",  # Overlay found on Brazilian sites
-            ".Campaign",
-        ]
-        target_str = ",".join(target_list)
-        javascript = f"""
-            document.querySelectorAll('{target_str}').forEach(el => el.remove());
-            var styleSheet = document.createElement('style');
-            styleSheet.innerText = '{target_str} {{ display: none !important; }}';
-            document.head.appendChild(styleSheet);
-         """
-        print("Executing common JavaScript")
-        page.evaluate(javascript)
-
-        # If there's custom javascript for this site, run it
-        custom_javascript = utils.get_javascript(site["handle"])
-        if custom_javascript:
-            print("Executing custom JavaScript")
-            try:
-                page.evaluate(custom_javascript)
-            except Error as error:
-                raise click.ClickException(error.message)
-
-        # Hide the scrollbars
-        print("Hiding scrollbars with CSS")
-        css = """document.body.style.overflow = 'hidden';"""
-        page.evaluate(css)
-
-        # Prevent Playwright from hovering over a link and highlighting it
-        print("Preventing mouse hovers")
-        css = """
-        const style = document.createElement("style");
-        style.innerHTML = "a:hover, a:focus { color: initial; text-decoration: initial; }";
-        document.head.appendChild(style);
-        """
-        page.evaluate(css)
-
-        # Give it another beat
-        print(f"Waiting {wait_seconds} seconds")
-        time.sleep(wait_seconds)
-
         # Take the screenshot
-        file_path = str(output_path / f"{site['handle'].lower()}.jpg")
-        print(f"Saving image to {file_path}")
+        jpeg_file_path = str(output_path / f"{site['handle'].lower()}.jpg")
```

Comment:
Why do we need to wrap this in `str()`?

Reply:
i'm not sure — I'm pretty sure this code (line 189) was doing this before I got here. Maybe if output_path is `None`? Or to satisfy typing?